packages/@lwc/engine-server/src/__tests__/fixtures/render-correct-closing-tags/modules/x/elements/elements.html

```diff
@@ -0,0 +1,19 @@
+<template>
```

Comment:
Add test cases for `class` attribute and `lwc:inner-html` on input or textarea or somthing.

Reply:
Adding `lwc:inner-html` to a void element like input throws an exception in engine server and renders nothing so I didn't add it. For v2 it doesn't throw but it doesn't render anything.

Reply:
We should open an issue to make it throw an error. `lwc:inner-html` doesn't make any sense for void elements, so throwing an error is a good behavior.

Reply:
Ok sounds good, I opened [this](https://github.com/salesforce/lwc/issues/5006) issue